docs(game/five): documentation for sv_replaceExeToSwitchBuilds flag (#493)
citizenfx/fivem-docs: content/docs/server-manual/server-commands.md

| 1436   | July 2021 update, includes new content from Blood Money DLC.      |
| 1491   | September 2022 update, limited content/changes.                   |

### `sv_replaceExeToSwitchBuilds [newValue]`

An experimental flag that determines how the client will run older game builds when requested with `sv_enforceGameBuild`. Either true or false.

> - true: the default value. Corresponds to the default behavior before this variable was introduced. The client will load executable and all the files corresponding to the enforced game build.

Reviewer: ```suggestion
- true: the default value. This will keep the current client behavior, the client will download all the files for the specific game build and have to restart their client to join servers.
```

I'm not entirely sure how this works, you should remove the "and have to restart their client to join servers." if this is still required for the new system

Author: Unfortunately reloads are still required.

> - false: The client will run executable for one of the latest game builds, regardless of which game build is enforced. Instead the set of content will be changed to correspond to the enforced game build (e.g. only relevant DLCs will be applied). From the user perspective the game will look the same way as under `sv_replaceExeToSwitchBuilds = true`.

Reviewer: ```suggestion
- false: The client will run executable for the latest game builds, regardless of which game build is enforced. Instead the client will only load the specific DLCs related to the game build specified in `sv_enforceGameBuild`. From the user perspective the game will look the same way as under `sv_replaceExeToSwitchBuilds = true`.
```

>     If set `sv_enforceGameBuild 1` - the client will run as if `sv_replaceExeToSwitchBuilds = false` regardless of which value is set. That's the only way how base game without DLCs behavior can be achieved.

Reviewer: ```suggestion
    When using `set sv_enforceGameBuild 1` the client will run as if `sv_replaceExeToSwitchBuilds = false` regardless of if it was set to `true` before. That's the only way how base game without DLCs behavior can be achieved.
```

Let me know if you agree with the wording here, it felt a bit hard to figure out what `which value is set` is referring to with context clues

Author: Suggested wording looks good. Applied. Thank you for the help! ^^

>     If `sv_enforceGameBuild = 1` - the client will run as if `sv_replaceExeToSwitchBuilds = false` regardless of which value is set. That's the only way how base game without DLCs behavior can be achieved.

Reviewer: ```suggestion
    If `set sv_enforceGameBuild 1` - the client will run as if `sv_replaceExeToSwitchBuilds = false` regardless of which value is set. That's the only way how base game without DLCs behavior can be achieved.
```

> NOTE: The difference in `sv_replaceExeToSwitchBuilds` should be **invisible** to players and server owners. If you notice a disparity please report it as a bug.

Reviewer: ```suggestion
NOTE: The difference in `sv_replaceExeToSwitchBuilds` should be **invisible** to players and server owners. If you notice a disparity please [report it as a bug](https://github.com/citizenfx/fivem/issues/new?assignees=&labels=bug%2Ctriage&projects=&template=bug_report.yml).
```
